cmd/database.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"xkcdcomics/model"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var Db *sql.DB
```

Comment:
The database object should be unexported and all access to it should only be done via exported functions from this package. This way we can be sure no other code makes db operations it isn't allowed to make.